alcs-frontend/src/app/shared/inline-editors/inline-textarea/inline-textarea.component.spec.ts

```diff
@@ -3,7 +3,7 @@ import { ComponentFixture, TestBed } from '@angular/core/testing';
 
 import { InlineTextareaComponent } from './inline-textarea.component';
 
-describe('InlineTextareaComponent', () => {
+describe('InlineTextComponent', () => {
```

Comment:
I think this should stay as InlineTextareaComponent ?

Reply:
done